src/components/runbooks/RunbookForm.js

```diff
@@ -0,0 +1,202 @@
+import React, {Component} from 'react';
+import {observable, action} from 'mobx';
+import {observer} from 'mobx-react';
+import {clone} from 'lodash';
+import cx from 'classnames';
+
+import FileInput from '../FileInput';
+import DownloadFileButton from '../DownloadFileButton';
+
+@observer
+export default class RunbookForm extends Component {
+  @observable key = Date.now()
```

Comment:
I'd name it somehow more meaningful and create a separate action with meaningful name to update it. It took me about a minute to recall what this hack is for :)